docs/index.md

```diff
@@ -11,19 +11,18 @@ OWT Android requires using Android SDK API level 16 or above. In order to use ha
 The following devices have been validated with our SDK:
 
    + Huawei* Mate9 (Android 7.0)
+   + Huawei* Mate20 (Android 9.0)
```

Comment:
Please keep this list in alphabet order, i.e.: put mate 20 before meta 9 because 2 is less than 9.

Reply:
done